addons/sale_stock/models/sale_order.py

```diff
@@ -14,10 +14,10 @@
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
-    incoterm = fields.Many2one(
-        'account.incoterms', 'Incoterm',
+    incoterms = fields.Many2one(
```

Comment:
Take the opportunity to use the correct pattern for m2o fields.
```suggestion
    incoterms_id = fields.Many2one(
```